### ISSUE TITLE
Window opts not working?

### ISSUE THREAD

**Issue by Oneechan69** [dzfrias/arena.nvim#14]
I tried this, as well as inside `opts` in my lazy plugin config but it didn't work:

```lua
require("arena").setup({
		max_items = 99,
		window = {
			height = 20
		}
})
```

**Comment by qmrodgers** [comment]
Submitted a pull request for this and an issue i encountered on windows PCs
#15

**Comment by dzfrias** [comment]
Should be closed by #15
